functions/index.js

```diff
@@ -14,6 +15,7 @@ app.use(cors());
 app.use(morgan('dev'));
 //Use Routes Here
 app.use('/v1/auth', users);
+app.use('/v1', routes);
```

Comment:
I think `routes` should be renamed descriptively to `locations` for easy identification just like `users`

Reply:
Ok i will update that now

Reply:
No `routes` is the correct here. `users` is what is wrong. It should be called `auth` or `authRoutes`